setup.py

```diff
@@ -73,6 +73,7 @@ def run(self):
     "win-unicode-console>=0.5; sys_platform == 'win32'",
     "pywin32>=225; sys_platform == 'win32'",
     "networkx>=2.1",
+    "psutil>=5.8.0",
```

Comment:
We should also remove https://github.com/iterative/dvc/blob/fa5487ebfe7dec9d035a3e140b326042bad345b6/dvc/info.py#L18 and related logic, since psutil will now be always present.